service_msgs/README.md

```diff
@@ -0,0 +1,13 @@
+# service_msgs
+This package contains message types useful for ros2 services.
+
+More information about services can be found on its [design article](http://design.ros2.org/articles/actions.html).
+
+For more information about ROS 2 interfaces, see [docs.ros.org](https://docs.ros.org/en/rolling/Concepts/About-ROS-Interfaces.html).
+
+## Messages (.msg)
+* [ServiceEventInfo](msg/ServiceEventInfo.msg): Service event information.
```

Comment:
I would elaborate here a bit more about how this is part of the service introspection feature (with a link to the REP).

Reply:
[f274555](https://github.com/ros2/rcl_interfaces/pull/143/commits/f27455529315fbbc6f49a16f852156e379bab2c5)